starter-code/index.html

```diff
@@ -119,18 +120,22 @@
     <script src="/vendor/scripts/html5sql.js"></script>
     <script src="/vendor/scripts/webdb.js"></script>
     <script src="/vendor/scripts/page.js"></script>
+    <script src="/vendor/scripts/key.js"></script>
```

Comment:
This is a semantic thing, but a token file isn't really a vendor file. I would personally put it inside of my model folder.